### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by jasheal** [ben-rogerson/twin.macro#676] [Q&A] [answered]
Hey! When can a new release be expected? I’m noticing a bunch of colour classes (and maybe others) from the latest version of tailwind not supported yet.

**Comment by ben-rogerson** [comment, marked as answer]
Hey there 👋 
The new features are being tested then a twin release release shortly after.
Follow the [tailwind v3 updates](https://github.com/ben-rogerson/twin.macro/issues/589) issue.

The colour issue is fixed by temporarily importing the color package into your tailwind config:

![image](https://user-images.githubusercontent.com/21288568/166079644-50906f91-3656-453f-a7f1-d5e9d8a427cc.png)
